### DISCUSSION TITLE
[Release] Yellowpill

### DISCUSSION THREAD

**Discussion by toiletman74** [nzp-team/nzportable#255] [Map Releases]
![78f0f50285dd11bef4946bc47283e49281-pills-lede](https://user-images.githubusercontent.com/91436591/226062085-42198dff-afc5-4ca2-ba05-f6e9b172c873.png)
They say that the gateway
to the real world is in
the hit game from the 80s
pac-man.
Are you in?

Map preview:
The layout looks like this 
![image](https://user-images.githubusercontent.com/91436591/226062569-cc5b5958-387d-4d2d-9942-ccdf0d06eb63.png)
![image](https://user-images.githubusercontent.com/91436591/229969359-f59cf921-ad12-4c8a-92b4-83f0cb3c71c6.png)

# TAKE THE YELLOWPILL
# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/11154750/yellowpill.zip)
[(This is a version with no dogs for if you get bugs with dogs)](https://github.com/nzp-team/nzportable/files/11154753/yellowpill.No.Dogs.zip)

**Comment by MotoLegacy** [comment]
This is a really cool, challenging concept! It would be awesome if you had lights placed around, that way its easier to tell where there are/aren't walls

**Comment by toiletman74** [comment]
Here's two updated versions of the map one with dogs and one without. All of the changes are purely cosmetic.
Special thanks to
CypressImplex#8684
&
Shrimp#4542
for helping me with some constructive criticism on how to make the map look better.

**Comment by toiletman74** [comment]
Here's a new update, fixed an issue for some platforms where zombies would float.
Thanks to bonniegamer202#2599 for pointing out the issue to me!

**Comment by toiletman74** [comment]
Realized that I didn't update the waypoint files when I released the last version so I updated it again with the new waypoint files

**Reply by Ellis243454** [reply]
Hey I have no clue why I would love to play your map but there is no wall textures on the 3ds

**Reply by Paindeluxe** [reply]
I am looking at this soon I‘ll give you a patched version if I can (:

**Comment by Jeyphr** [comment]
Check my review here: https://youtu.be/Id787ymxY80

**Comment by diegops2pro-dot** [comment]
Doesn't work on psp you can't move and wall textures don't work
